docs/interchain-security/proto-docs.md

```diff
@@ -1,518 +0,0 @@
-<!-- This file is auto-generated. Please do not modify it yourself. -->
```

Comment:
This is auto-generated. This is good to have.

Reply:
We could use something like https://github.com/pseudomuto/protoc-gen-doc. But I'm having trouble understanding what that documentation would be used for, since `.proto` files are already pretty self explanatory imo